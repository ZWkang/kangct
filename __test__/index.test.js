import ALL from '../src/index';


describe('The Entry Should Be Right', ()=>{
    it('here export should be a object', ()=>{
        expect(ALL).toHaveProperty('Button')
    })
})