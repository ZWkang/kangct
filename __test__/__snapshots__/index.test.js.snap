// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`kangct project entry export snapshot 1`] = `
Array [
  "Accordion",
  "Collapse",
  "Alert",
  "Animate",
  "Animation",
  "Badge",
  "BorderMagicCom",
  "Box",
  "Breadcrumb",
  "Button",
  "Card",
  "Column",
  "Dialog",
  "Dropdown",
  "ErrorBoundaries",
  "Flex",
  "FlexItem",
  "H1",
  "H2",
  "H3",
  "H4",
  "H5",
  "H6",
  "Abbr",
  "Del",
  "Mark",
  "Ol",
  "Ul",
  "Sup",
  "Sub",
  "A",
  "Ins",
  "Hr",
  "Strong",
  "B",
  "Pre",
  "Code",
  "Figcaption",
  "Small",
  "P",
  "Li",
  "Blockquote",
  "Hero",
  "HeroBody",
  "HeroA",
  "HeroContainer",
  "HeroH1",
  "HeroLeft",
  "HeroRight",
  "HeroP",
  "ImageComponent",
  "Lifecycle",
  "Loading",
  "Pagination",
  "Panel",
  "Progress",
  "Protal",
  "Rect",
  "ScrollTo",
  "Tab",
  "Table",
  "Toggle",
  "ToolTips",
  "LazyLoad",
  "Intl",
]
`;
